iceoryx_posh/include/iceoryx_posh/runtime/posh_runtime.hpp

```diff
@@ -47,12 +47,12 @@ class NodeData;
 enum class FindServiceError
 {
     INVALID_STATE,
-    UNABLE_TO_WRITE_TO_ROUDI_CHANNEL,
-    INSTANCE_CONTAINER_OVERFLOW
+    UNABLE_TO_WRITE_TO_ROUDI_CHANNEL, /// @todo #415 remove as IPC channel won't be used
```

Comment:
Can we remove `FindServiceError` directly in here? I am asking since you already created the container with the size and with that the todo is already done?!

Reply:
Will be done in a follow-up PR, already a todo in #415